webpack.config.development.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 
+var _ = require('lodash');
 var webpack = require('webpack');
 var baseConfig = require('./webpack.config.base');
 
-var config = Object.create(baseConfig);
-config.plugins = [
-  new webpack.optimize.OccurenceOrderPlugin(),
-  new webpack.DefinePlugin({
-    'process.env.NODE_ENV': JSON.stringify('development')
-  })
-];
-
-module.exports = config;
+module.exports = _.merge({}, baseConfig, {
```

Comment:
Using `_.merge` here instead of `Object.create(baseConfig)` because the `Object.create` route doesn't clone object references so changes to them change the `baseConfig` properties.